ui/src/components/Dashboard/index.jsx

```diff
@@ -188,8 +197,26 @@ export default class Dashboard extends Component {
     navOpen: false,
     showHelpView: false,
     error: null,
+    versionNumber: null,
   };
 
+  componentDidMount() {
+    this.loadVersionNumber();
+  }
+
+  async loadVersionNumber() {
+    try {
+      const {
+        default: versionNumber,
+      } = await import(`../../../../.git-version`);
+
+      this.setState({ versionNumber });
```

Comment:
Set the version in `componentDidMount`. `loadVersionNumber` should just return the value.